src/components/DomainItem/DomainItem.js

```diff
@@ -128,18 +127,9 @@ const Label = ({ domain, isOwner }) => {
     case 'Open':
       text = 'Available'
       break
-    case 'Auction':
-      text = 'Bidding Period'
-      break
```

Comment:
Are you getting rid of `Auction`, `Forbidden`, and `Reveal` because we already migrated into the permanent registrars? If so I suggest to clean up other parts like below

<img width="307" alt="Screenshot 2019-08-14 at 15 04 40" src="https://user-images.githubusercontent.com/2630/63027437-dfca4580-bea4-11e9-8c34-c966a2974b1a.png">

Reply:
Agreed. I'll clean this up as part of my other PR, which will follow this as it requires some refactoring of code that I will get rid of as well.